vpr/src/place/initial_placement.cpp

```diff
@@ -147,6 +147,26 @@ static bool is_loc_legal(const t_pl_loc& loc,
                          const PartitionRegion& pr,
                          t_logical_block_type_ptr block_type);
 
+/**
+ * @brief  Helper function to choose a subtile in specified location if compatible and available one exits.
+ *  
+ *   @param centroid The centroid location at which the subtile will be selected using its x,y, and layer.
+ *   @param block_type Logical block type of the macro head member.  
+ *   @param block_loc_registry Placement block location information. To be filled with the location
+ *   where pl_macro is placed.
```

Comment:
I think you should remove the "To be filled with the location where pl_macro is placed."
I'd just say:
"Information on where other blocks have been placed."